### DISCUSSION TITLE
Camo cache cannot be flushed?

### DISCUSSION THREAD

**Discussion by UlyssesZh** [anuraghazra/github-readme-stats#3485] [Q&A] [answered]
The camo cache on my language card cannot be flushed. I tried using the cameo flush API, no use.

You can go to my profile and see the displayed language card. You can inspect the HTML source to see the actual URL, and the result of this URL is different from the displayed one. It has been like that for weeks.

I also edited the link to the language card in the Markdown source, and the language card is still shown as the old one. If I open the link directly, I can see that the contents did change, but it was just the camo cache that held the old image.

What do I do now?

**Comment by qwerty541** [comment, marked as answer]
Hey, @UlyssesZh! I achieved this result by replacing comma with %23 in your URL, querystring parameters content should be url encoded in this case. Looks like this error in URL was the reason of broken cache on GitHub CDN.

```html
<picture>
  <source
    srcset="https://github-readme-stats.vercel.app/api?username=UlyssesZh&show_icons=true&count_private=true&hide_title=true&include_all_commits=true&hide_rank=true&number_format=long&hide_title=true&show=discussion_started&hide_border=true&theme=dark"
    media="(prefers-color-scheme: dark)"
  />
  <source
    srcset="https://github-readme-stats.vercel.app/api?username=UlyssesZh&show_icons=true&count_private=true&hide_title=true&include_all_commits=true&hide_rank=true&number_format=long&hide_title=true&show=discussion_started&hide_border=true"
    media="(prefers-color-scheme: light), (prefers-color-scheme: no-preference)"
  />
  <img src="https://github-readme-stats.vercel.app/api?username=UlyssesZh&show_icons=true&count_private=true&hide_title=true&include_all_commits=true&hide_rank=true&number_format=long&hide_title=true&show=discussion_started&hide_border=true" />
</picture>

<picture>
  <source
    srcset="https://github-readme-stats.vercel.app/api/top-langs/?username=UlyssesZh&layout=compact&hide_title=true&hide=tex%23html&langs_count=16&hide_border=true&theme=dark"
    media="(prefers-color-scheme: dark)"
  />
  <source
    srcset="https://github-readme-stats.vercel.app/api/top-langs/?username=UlyssesZh&layout=compact&hide_title=true&hide=tex%23html&langs_count=16&hide_border=true"
    media="(prefers-color-scheme: light), (prefers-color-scheme: no-preference)"
  />
  <img src="https://github-readme-stats.vercel.app/api/top-langs/?username=UlyssesZh&layout=compact&hide_title=true&hide=tex%23html&langs_count=16&hide_border=true" />
</picture>
```

<picture>
  <source
    srcset="https://github-readme-stats.vercel.app/api?username=UlyssesZh&show_icons=true&count_private=true&hide_title=true&include_all_commits=true&hide_rank=true&number_format=long&hide_title=true&show=discussion_started&hide_border=true&theme=dark"
    media="(prefers-color-scheme: dark)"
  />
  <source
    srcset="https://github-readme-stats.vercel.app/api?username=UlyssesZh&show_icons=true&count_private=true&hide_title=true&include_all_commits=true&hide_rank=true&number_format=long&hide_title=true&show=discussion_started&hide_border=true"
    media="(prefers-color-scheme: light), (prefers-color-scheme: no-preference)"
  />
  <img src="https://github-readme-stats.vercel.app/api?username=UlyssesZh&show_icons=true&count_private=true&hide_title=true&include_all_commits=true&hide_rank=true&number_format=long&hide_title=true&show=discussion_started&hide_border=true" />
</picture>

<picture>
  <source
    srcset="https://github-readme-stats.vercel.app/api/top-langs/?username=UlyssesZh&layout=compact&hide_title=true&hide=tex%23html&langs_count=16&hide_border=true&theme=dark"
    media="(prefers-color-scheme: dark)"
  />
  <source
    srcset="https://github-readme-stats.vercel.app/api/top-langs/?username=UlyssesZh&layout=compact&hide_title=true&hide=tex%23html&langs_count=16&hide_border=true"
    media="(prefers-color-scheme: light), (prefers-color-scheme: no-preference)"
  />
  <img src="https://github-readme-stats.vercel.app/api/top-langs/?username=UlyssesZh&layout=compact&hide_title=true&hide=tex%23html&langs_count=16&hide_border=true" />
</picture>

**Reply by UlyssesZh** [reply]
Thank you. That's the reason. Just need to point out that comma is `%2C` instead of `%23`.
